railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt

```diff
@@ -51,6 +51,11 @@
 # and asynchronous queries will then be enabled.
 # Rails.application.config.active_support.executor_around_test_case = true
 
+# Define the isolation level of most of Rails internal state.
+# If you use a fiber based server or job processor, you should set it to `:fiber`.
+# Otherwise it should be set to `:thread`.
```

Comment:
When reading this it makes me think, why isn't this set? And what's the default given that Rails must maintain a lot of internal state? I'm most likely on puma so why isn't this set to `:thread`? I think it's unclear what this does and why as a user it would make sense to enable this, but I don't have a concrete suggestion right now.